claims.go

```diff
@@ -81,8 +81,8 @@ func (c *StandardClaims) VerifyIssuer(cmp string, req bool) bool {
 	return verifyIss(c.Issuer, cmp, req)
 }
 
-// VerifyNotBefore compares the nbf claim against cmp.
-// If required is false, this method will return true if the value matches or is unset
+// VerifyNotBefore compares the exp claim against cmp (cmp >= nbf).
```

Comment:
```suggestion
// VerifyNotBefore compares the nbf claim against cmp (cmp >= nbf).
```

One more :) then we are good to go.